design-reviews/recipes/task-design-review-template.md

```diff
@@ -0,0 +1,47 @@
+# Your Task Design Title Here (prefix with DRAFT/WIP to indicate level of completeness)
+
+> When developing a design document for a new task, it should contain a detailed design proposal demonstrating how it will solve the goals outlined below.  
+> Not all tasks require a design review, but when they do it is likely that there many unknowns or the solution may be more complex.
+> The design should include diagrams, pseudo code, interface contracts as needed to provide a detailed understanding of the proposal.
+
+* [Task Name](http://link-to-task-work-item)
+* [Story Name](http://link-to-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview / Problem Statement
+
+* Describe the task with a high-level summary.
+* Consider additional background and justification, for posterity and historical context.
+
+## Goals / In-Scope
```

Comment:
I would repurpose this section to be put more terms of a directional decision that needs to be made. For example... 

## Decision
In order to complete this task, we need to make a decision on the api contract for the new POST endpoint being added with this task. The contract must provide to the consuming service

### Scope
* An identifier that can be used to look up the created resource
* Status indicator if the downstream processing completed

I might templatize that as...

## Decision
[`TODO: Summarize the decision that needs to be made before we can complete this task`]

### Scope
[`TODO: Add bullet list of what this task must fulfill to make the story as a whole pass acceptance`]

Thoughts @fnocera @arsenvlad

Reply:
I agree with the suggestion to add 'decision', but it seemed repetitive to the proposed options and scope would clearly define the goal, so had to lean towards the previous template over this approach.